pages/common/openssl-ts.md

```diff
@@ -0,0 +1,20 @@
+# openssl ts
+
+> OpenSSL command to generate and verify timestamps.
+> More information: <https://www.openssl.org/docs/manmaster/man1/openssl-ts.html>.
+
+- Generate a SHA-512 timestamp request of `file` (output to `file.tsq`):
+
+`openssl ts -query -data {{path/to/file}}  -sha512  -out {{path/to/file.tsq}}`
+
+- Check the date and metadata of a specific timestamp response file:
+
+`openssl ts -reply -in {{path/to/file.tsr}} -text`
+
+- Verify timestamp request `file.tsq` and timestamp response `file.tsr` from the server with ssl certificate `cert.pem`:
+
+`openssl ts -verify -in {{path/to/file.tsr}} -queryfile {{path/to/file.tsq}} -partial_chain -CAfile {{path/to/cert.pem}}`
+
+- Create a timestamp response for request `file.tsq` using key `tsakey.pem` and signing certificate `tsacert.pem` (output to `file.tsr`):
```

Comment:
```suggestion
- Create a timestamp response for request using key and signing certificate and output to `file.tsr`:
```
Leave this one for others

Reply:
I'd say
```suggestion
- Create a timestamp response for request using key and signing certificate, and output it to `file.tsr`:
```

Reply:
`- Create a timestamp response for request using key and signing certificate, and output the timestamp to `file.tsr`:`

What about?

Reply:
sgtm 👍🏻

Reply:
the `it` is required however the comma isn't needed

Reply:
> the `it` is required however the comma isn't needed

So
` - Create a timestamp response for request using key and signing certificate and output it to `file.tsr`:`

Reply:
Yes

Reply:
>the `it` is required however the comma isn't needed

I don't really agree, but @0x1080's [latest](https://github.com/tldr-pages/tldr/pull/6125#discussion_r652132016) looks good to me; let's go with it.